src/Raven.Studio/typescript/viewmodels/manage/certificates.ts

```diff
@@ -471,6 +479,12 @@ class certificates extends viewModelBase {
 
                 switch (model.mode()) {
                     case "generate":
+                    case "reGenerate":
+                        
+                        if (model.deleteExpired()) {
```

Comment:
I wonder if we should delete after we generate new one. Even button had such text some time ago. If generate fails don't delete.

Reply:
done